module Util where

import Control.Exception (try)
import Control.Applicative

